Content.Server/GameTicking/GameTicker.cs

```diff
@@ -260,16 +264,20 @@ public void SetStartPreset(Type type)
             UpdateInfoText();
         }
 
-        private IEntity _spawnPlayerMob()
+        private IEntity _spawnPlayerMob(Job job)
         {
             var entity = _entityManager.SpawnEntityAt(PlayerPrototypeName, _getLateJoinSpawnPoint());
             if (entity.TryGetComponent(out InventoryComponent inventory))
             {
-                var uniform = _entityManager.SpawnEntity("UniformAssistant");
-                inventory.Equip(EquipmentSlotDefines.Slots.INNERCLOTHING, uniform.GetComponent<ClothingComponent>());
+                var gear = _prototypeManager.Index<StartingGearPrototype>(job.StartingGear).Equipment;
 
-                var shoes = _entityManager.SpawnEntity("ShoesBlack");
-                inventory.Equip(EquipmentSlotDefines.Slots.SHOES, shoes.GetComponent<ClothingComponent>());
+                foreach (var slotEquipementPair in gear)
+                {
+                    EquipmentSlotDefines.Slots slot;
+                    Enum.TryParse(slotEquipementPair.Key.ToUpper(), out slot);
+                    var equipement = _entityManager.SpawnEntity(slotEquipementPair.Value);
+                    inventory.Equip(slot, equipement.GetComponent<ClothingComponent>());
```

Comment:
Minor style suggestion, these should probably be "equipment" instead of "equipement" to match the existing fields/variables.